stable/sentry/values.yaml

```diff
@@ -177,3 +177,34 @@ redis:
 config:
   configYml: ""
   sentryConfPy: ""
+
+## Prometheus Exporter / Metrics
+##
+metrics:
+  enabled: false
+
+  ## Metrics exporter resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  # resources:
+  #   limits:
+  #     cpu: 100m
+  #     memory: 100Mi
+  #   requests:
+  #     cpu: 100m
+  #     memory: 100Mi
+
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  # schedulerName:
+  # Optional extra labels for pod, i.e. redis-client: "true"
+  # podLabels: []
+  service:
+    type: ClusterIP
+    labels:
+      monitor: me
```

Comment:
This looks strange. Do we need it by default?

Reply:
No you're right, it seems to work without it so I have removed it.